src/main/scala-2.12/org/ergoplatform/appkit/ergotool/dex/CreateBuyOrderCmd.scala

```diff
@@ -89,22 +89,40 @@ case class CreateBuyOrderCmd(toolConf: ErgoToolConfig,
 
 
 object CreateBuyOrderCmd extends CmdDescriptor(
-  name = "dex:BuyOrder", cmdParamSyntax = "<wallet file> <buyerAddr> <ergAmount> <tokenId> <tokenAmount>, <dexFee>",
+  name = "dex:BuyOrder", cmdParamSyntax = "<storageFile> <buyerAddr> <ergAmount> <tokenId> <tokenAmount>, <dexFee>",
   description = "put a token buyer order with given <tokenId> and <tokenAmount> to buy at given <ergPrice> price with <dexFee> as a reward for anyone who matches this order with a seller, with <buyerAddr> to be used for withdrawal \n " +
-    "with the given <wallet file> to sign transaction (requests storage password)") {
+    "with the given <storageFile> to sign transaction (requests storage password)") {
+
+  override val parameters: Seq[CmdParameter] = Array(
+    CmdParameter("storageFile", FilePType,
+      "storage with secret key of the sender"),
+    CmdParameter("storagePass", SecretStringPType,
+      "password to access sender secret key in the storage", None,
+      Some(ctx => ctx.console.readPassword("Storage password>"))),
+    CmdParameter("buyerAddr", AddressPType,
+      "address of the buyer"),
+    CmdParameter("ergAmount", LongPType,
+      "amount of NanoERG to pay for tokens"),
+    CmdParameter("tokenId", ErgoIdPType,
+      "token id to buy"),
+    CmdParameter("tokenAmount", LongPType,
+      "token amount to buy"),
+    CmdParameter("dexFee", LongPType,
+      "reward for anyone who matches this order with seller's order")
```

Comment:
Why not copy all descriptions from ScalaDoc above. They are eccentialy the same parameters.
Since this parameters is exactly a metadata for the constructor arguments.
You may need to unify descriptions, and make them readable in both contexts.

Reply:
Done.